### PR TITLE
Feature/new auto registration

### PR DESCRIPTION
**Pull request by PedroTroller**

### CODE REVIEW

File: spec/Knp/Rad/AutoRegistration/DefinitionBuilder/FormExtensionBuilderSpec.php [hunk 1]
```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace spec\Knp\Rad\AutoRegistration\DefinitionBuilder;
+
+use Knp\Rad\AutoRegistration\Finder\BundleFinder;
+use Knp\Rad\AutoRegistration\Kernel\KernelWrapper;
+use Knp\Rad\AutoRegistration\Reflection\ClassAnalizer;
```
Comment (akerouanton) [inline spec/Knp/Rad/AutoRegistration/DefinitionBuilder/FormExtensionBuilderSpec.php]: `Analyzer` ;)

File: spec/Knp/Rad/AutoRegistration/Finder/BundleFinderSpec.php [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+use PhpSpec\ObjectBehavior;
+
+class BundleFinderSpec extends ObjectBehavior
+{
+    public function it_is_initializable()
+    {
+        $this->shouldHaveType('Knp\Rad\AutoRegistration\Finder\BundleFinder');
+    }
+}
```
Comment (akerouanton) [inline spec/Knp/Rad/AutoRegistration/Finder/BundleFinderSpec.php:13]: No spec here ?

File: spec/Knp/Rad/AutoRegistration/ServiceNameGenerator/BundleServiceNameGeneratorSpec.php [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    {
+        $kernel->getBundles()->willReturn([$bundle1, $bundle2]);
+
+        $this->beConstructedWith($kernel, $generator);
+
+        $bundle1->getNamespace()->willReturn('Company\MyBundle');
+        $bundle1->getName()->willReturn('bundle1');
+
+        $bundle2->getNamespace()->willReturn('Company\MyOther\Bundle');
+        $bundle2->getName()->willReturn('Bundle2');
```
Comment (akerouanton) [inline spec/Knp/Rad/AutoRegistration/ServiceNameGenerator/BundleServiceNameGeneratorSpec.php:22]: Why `bundle1` and `Bundle2` ?

Comment (PedroTroller) [inline spec/Knp/Rad/AutoRegistration/ServiceNameGenerator/BundleServiceNameGeneratorSpec.php:22]: Why not....

File: README.md [hunk 4]
```diff
@@ -47,3 +48,23 @@ Now all repositories are auto-registred.
...
+    services:
+        form_type: true
+        form_type_extension: true
+```
+
+Now all form types and form type extensions are auto-registred.
+
+###Requirements
+
+Your form types and form type extensions should be stored under `Form` sub-namespace.
```
Comment (akerouanton) [inline README.md:71]: There is nothing about the new `bundles` parameter.

Comment (PedroTroller) [inline README.md:71]: Done

**Pull request merged**